src/components/EventDetail/SeatReservation/SeatMapSelection/index.tsx

```
@@ -184,149 +189,239 @@ const SeatMapSelection = ({
+                  >
+                    <option value="none" selected disabled hidden>
+                      Select a date
+                    </option>
+                    {event.schedules.map((schedule, index) => (
+                      <>
+                        {schedule.date && (
+                          <option key={index} value={schedule.date}>
+                            {dateFnsFormat(new Date(schedule.date), 'dd/MM/yyyy')}
+                          </option>
+                        )}
+                      </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)